src/Forms/RequiredFields.php

```diff
@@ -11,6 +12,8 @@
  *
  * Validation is performed on a field by field basis through
  * {@link FormField::validate}.
+ *
+ * @deprecated 5.4.0 Will be renamed to SilverStripe\Forms\RequiredFieldsValidator
```

Comment:
Should we move the form validator classes to a `SilverStripe\Forms\Validation` namespace while we're at it, or nah?

Reply:
Yeah I think it is, will update